pages/button.html

```diff
@@ -0,0 +1,140 @@
+                <button class="button inverted-warning">Warning</button>
+                <button class="button inverted-danger">Danger</button>
+            </div>
+            <pre class="component-code-snippet">
+                <code class="language-html">
+                    <button class="button inverted-primary">Primary</button>
+                    <button class="button inverted-secondary">Secondary</button>
+                    <button class="button inverted-success">Success</button>
+                    <button class="button inverted-info">Info</button>
+                    <button class="button inverted-warning">Warning</button>
+                    <button class="button inverted-danger">Danger</button>
+                </code>
+            </pre>
+        </section>
+
+        <section class="component-section">
+            <h4>Disabled Buttons</h4>
+            <p>When you don't want a user to click on a button before performing some action, just add a class <span class="secondary-text"> disabled</span> to the button</p>
+            <div class="component-example-container">
+                <button class="button disabled">Disabled</button>
+                <button class="button primary disabled">Primary</button>
+                <button class="button inverted-danger disabled">Disabled</button>
+            </div>
+            <pre class="component-code-snippet">
+                <code class="language-html">
+                    <button class="button disabled">Disabled</button>
+                    <button class="button primary disabled">Primary</button>
+                    <button class="button inverted-danger disabled">Disabled</button>
+                </code>
+            </pre>
+        </section>
+
+        <section class="component-section">
+            <h4>Icon Buttons</h4>
+            <p>You can add icons as buttons as well. They are compact and are widely used in websites.</p>
+            <div class="component-example-container">
+                <i class="fas fa-download button"></i>
+                <i class="fas fa-share-alt button inverted-info"></i>
+            </div>
+            <pre class="component-code-snippet">
+                <code class="language-html">
+                    <i class="fas fa-download button"></i>
+                    <i class="fas fa-share-alt button inverted-info"></i>
+                </code>
+            </pre>
+        </section>
+
+        <section class="component-section">
+            <h4>Floating Buttons</h4>
+            <p>Floating button is fixed at the bottom-right corner. You customize it by adding different color and icon.</p>
+            <div class="component-example-container">
+                <i class="fas fa-plus button danger floating"></i>
+            </div>
+            <pre class="component-code-snippet">
+                <code class="language-html">
+                    <img class="img-responsive img-squared" src="/images/cup1.jpg" alt="responsive-image"/>
```

Comment:
there should be floating icon code i guess

Reply:
Ohh! Thanks. I have changed it

Reply:
looks good now